doc/classes/RenderingDevice.xml

```diff
@@ -2044,28 +2044,34 @@
 		<constant name="DYNAMIC_STATE_STENCIL_REFERENCE" value="64" enum="PipelineDynamicStateFlags" is_bitfield="true">
 		</constant>
 		<constant name="INITIAL_ACTION_CLEAR" value="0" enum="InitialAction">
+			Start rendering and clear the whole framebuffer, region or not (supply appropriate parameters when using this action).
 		</constant>
 		<constant name="INITIAL_ACTION_CLEAR_REGION" value="1" enum="InitialAction">
+			Start rendering and clear the framebuffer in the specified region (supply appropriate parameters when using this action).
```

Comment:
```suggestion
			Start rendering and clear the framebuffer in the specified region.
```
``(supply appropriate parameters when using this action)`` sounds a little too cryptic when the "appropriate parameter" is the "region" which we explicitly tell the user to specify in the main sentence. I would just remove the parenthetical here as it may confuse users and it doesn't technically add any information